gulp-tasks/documentation.js

```diff
@@ -54,6 +54,11 @@ const documentPackage = (projectPath) => {
 
     handleBarsData.hasRefDocs = (projectMetadata.name !== 'sw-cli');
 
+    // sw-cli needs a custom install command
+    if (projectMetadata.name === 'sw-cli') {
+      handleBarsData.installCmd = `npm install -g sw-cli`;
+    }
+
```

Comment:
One ask: can we change this to `npm install --global sw-cli`? Experienced Node folks totally get the `-g` shorthand but those more new to it could benefit from knowing what we're doing here.